docs/source/configuration.rst

```diff
@@ -679,6 +679,17 @@ Data in localStorage on the other hand is kept indefinitely.
     roster contact statuses will not become out of sync in a single session,
     only across more than one session.
 
+synchronize_presence
+--------------------
```

Comment:
Looking at the spec again, they call it your "availability status". In converse.js I've been calling it "chat status". What makes it more confusing is that there is also a `<status>` element in some `<presence>` stanzas which is the status message that people can write, like "I'm going to town for a bit, will be back later".

In any case, I'd like to keep things as clear and distinct as possible, and if we can keep it closer to the spec that's also better.

So I think `synchronise_availability` is a better name here than `synchronize_presence`.